examples/dataset/face_recognition_11/face_recognition_11/upload_dataset.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from argparse import ArgumentParser
+from argparse import Namespace
+
+import pandas as pd
+from face_recognition_11.constants import DATASET
+from face_recognition_11.constants import DATASET_DETECTION
+from face_recognition_11.constants import DATASET_METADATA
+from face_recognition_11.constants import DATASET_PAIRS
+from face_recognition_11.constants import TASK
+
+from kolena.annotation import BoundingBox
+from kolena.annotation import Keypoints
+from kolena.asset import ImageAsset
+from kolena.dataset import upload_dataset
+
+
+def run(args: Namespace) -> None:
+    df_metadata = pd.read_csv(DATASET_METADATA)
+    df_pairs = pd.read_csv(DATASET_PAIRS)
+    df_detection = pd.read_csv(DATASET_DETECTION)
+
+    genuine_pairs = {}
+    imposter_pairs = {}
+    for record in df_pairs.itertuples():
+        if record.is_match:
+            genuine_pairs[record.locator_1] = record.locator_2
```

Comment:
Should we rename these locators to `locator_left` and `locator_right` to be consistent with the left / right usage?